subjects/java/piscine/raids/Jaikin/audit/README.md

```diff
@@ -0,0 +1,49 @@
+#### Functional
+
+- Execute the program using the command `Java Main`.
```

Comment:
change this:
- Execute the program using the command `Java Main`.
- Verify if it compiles and runs without any warnings.
with this :
- Try to run the program
- Does the program compile and run without any error or warning?

Reply:
It's done